### PR TITLE
Add more code examples to tutorial

### PR DESCRIPTION
**Pull request by f**

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -49,11 +57,32 @@ get "/" do
 end
 ```
 
-## Run your awesome web project.
+All the code should look like this:
+
+```ruby
+require "./crystal_test/*"
```
Comment (sdogruyol) [inline docs/tutorial.md]: Buna gerek yok

File: docs/tutorial.md [hunk 2]
```diff
@@ -49,11 +57,32 @@ get "/" do
...
+  end
+end
+```
+
+## 5. Run your awesome web project.
 
 ```
 crystal build --release src/awesome_web_project.cr
 ./awesome_web_project
 ```
```
Comment (ilatif) [inline docs/tutorial.md]: I think this should be mentioned only for `production` mode. In `development` environment this seems a bit tedious. Having run `crystal src/awesome_web_project.cr` works equally fine and should be encouraged for `development` environment.

Comment (sdogruyol) [inline docs/tutorial.md]: Yes, i agree on this

Comment (ilatif) [inline docs/tutorial.md]: Thanks :+1:.

Comment (ilatif) [inline docs/tutorial.md]: I think we should also mention magic param `_method` explicitly to make documentation more clearer.

**Pull request merged**